utils/group-emails-to-lowercase.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python group-emails-to-lowercase.py -group MyConf.org/2017/Reviewers --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# Sets all members that are emails to lowercase.  Notifies user if there are profiles or groups using the
+# uppercase letters in the
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('-g','--group', required=True, help = "the full path of the group ex. MyConf.org/2017")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+def get_profile(value):
+    profile = None
+    try:
+        profile = client.get_profile(value)
+    except openreview.OpenReviewException as e:
+        # throw an error if it is something other than "not found"
+        if e[0][0] != 'Profile not found':
+            print "OpenReviewException: {0}".format(e)
+            raise e
+    return profile
+
+def get_group(value):
+    group = None
+    try:
+        group = client.get_group(value)
+    except openreview.OpenReviewException as e:
+        # throw an error if it is something other than "not found"
+        if e[0][0]['type'] != 'Not Found':
+            print "OpenReviewException: {0}".format(e)
+            raise e
+    return group
+
+# return True if it finds an uppercase letter
+def has_uppercase(value):
+    has_upper = False
+    for character in value:
+        if character.isupper():
+            has_upper = True
+    return has_upper
+
+# locate emails w/ uppercase letters
+#   replace with lowercase email
+def upper_to_lower(group_name):
+    target_group = client.get_group(group_name)
```

Comment:
here you are not using your internal function, why? what happens if the group_name doesn't exist?